Automattic/wp-calypso
# DIFM Content Form: add character limit warning

Fixes https://github.com/Automattic/martech/issues/2728
Fixes https://github.com/Automattic/martech/issues/2729

## Proposed Changes

* Adds a character counter to the text area fields on the DIFM content submission form.
* Also adds a warning when the user exceeds the limit. The current limit is 5000 characters.

<img width="773" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/b264a3bb-c5d1-40fe-a5e6-1d68d41d4038">



## Testing Instructions



* Go to `/start/site-content-collection/website-content?siteSlug=<site slug>` for a site with a in-progress DIFM purchase.
* Confirm that the character counter is visible once you start typing.
* Confirm that you see the warning once you exceed 5000 characters.

## Pre-merge Checklist
